runtime/lua/vim/lsp/protocol.lua

```diff
@@ -942,5 +943,14 @@ protocol.Methods = {
   --- The `workspace/workspaceFolders` is sent from the server to the client to fetch the open workspace folders.
   workspace_workspaceFolders = 'workspace/workspaceFolders',
 }
+local function freeze(t)
```

Comment:
why would you add this here?
perhaps rebased on wrong branch?

Reply:
This was autogenerated when running `nvim -l scripts/gen_lsp.lua gen --methods`.

It is hardcoded in the script: https://github.com/neovim/neovim/blob/8f5e90811025bcc652146168d5116f8d73fe8f81/scripts/gen_lsp.lua#L114-L127

It seems someone forgot to run it in the past. For what I checked, every reference to lsp.Methods complies with this changes.

Reply:
🤦‍♂️ I didn't realize that this also was part of the autogenerated things

Reply:
freeze was removed in https://github.com/neovim/neovim/commit/0a0962a2e8d9ffa5f03e492fa293e964650a965e , please regen and rebase.

Reply:
Just completed the regen and rebase. A different set of changes was generated due to the regen script, but they seem appropriate.